subsys/bluetooth/services/fast_pair/fp_storage/fp_storage_ak.c

```diff
@@ -488,12 +1070,49 @@ static int fp_storage_ak_reset(void)
 		}
 	}
 
+	if (IS_ENABLED(CONFIG_BT_FAST_PAIR_STORAGE_AK_BOND)) {
+		FP_BONDS_FOREACH(bond) {
```

Comment:
Could we reuse `bond_remove` function? Maybe we could even always accept `-ESRCH` returned by `bt_request_cb->bond_remove` to simplify implementation on our end